hunt/app/models/animal.rb

```diff
@@ -1,2 +1,14 @@
 class Animal < ApplicationRecord
+  validates :name, presence: true
+  validates :category, presence: true
+  validates :state, presence: true
+  $CATEGORIES = ['Big Game','Small Game','Migratory Birds', 'Fur', 'Saltwater Fishing', 'Freshwater Fishing']
```

Comment:
In Ruby, variables beginning with `$` denote a global variable. This solution works, but we usually try not to use these in case they are somehow changed - this could affect the entire application. For some more reading you can look here: https://www.thoughtco.com/global-variables-2908384

Instead, this will work just fine without the `$` at the beginning. In order to access this constant in the view, you would use `Animal::STATES`. This also helps us see where this collection is coming from, and would be helpful in the case that we have multiple models that have different categories. The same goes for STATES below.